es-core/src/Sound.cpp

```diff
@@ -24,7 +29,7 @@ std::shared_ptr<Sound> Sound::getFromTheme(const std::shared_ptr<ThemeData>& the
 	LOG(LogInfo) << " req sound [" << view << "." << element << "]";
 
 	const ThemeData::ThemeElement* elem = theme->getElement(view, element, "sound");
-	if(!elem || !elem->has("path"))
+	if (!elem || !elem->has("path"))
```

Comment:
As above

Reply:
Reverted. Looking at the rest of this some of the conditionals have spaces. But I agree that readability between diffs is easier when there are not too many changes especially when it just trivial.